### ISSUE TITLE
adslzone.net (solved by generic filter)

### ISSUE THREAD

**Issue by timofonic** [reek/anti-adblock-killer#2187]
Hello.

adslzone.net finds the adblock. I use ublock Origin.

It's solved by generic filter:
@@||adslzone.net/*ad$script

(is it going to be needed forever?)

**Comment by rjkpa** [comment]
Yes, unless the site owner decides to remove the anti adblock or change the detecting mechanism.

**Comment by timofonic** [comment]
@rjkpa But are those filters going to be added to some list so future userss of AAK wouldn't need to do this manually?

**Comment by uBlock-user** [comment]
Unfortunately, AAK is dead as REEK has abandoned the project, so no.

**Comment by TPS** [comment]
>Unfortunately, AAK is dead as REEK has abandoned the project, so no.

@uBlock-user Citation needed, please. :pray:

**Comment by X01X012013** [comment]
Reek usually roll out an update every month, but he is gone for 4 months now. It is a LOT of work trying to maintain a project like this, I understand if he has other priorities. I just hope he will be back soon.

**Comment by TPS** [comment]
@X01X012013 That's why I posted....  I hope @reek is OK, but if he's just busy, good luck to him. :+1:

**Comment by TPS** [comment]
@Reek [lives!](https://github.com/reek/anti-adblock-killer/issues/2359#issuecomment-259325481)
